src/hotspot/share/opto/convertnode.hpp

```diff
@@ -213,6 +214,30 @@ class ConvL2INode : public ConvertNode {
   virtual Node* Ideal(PhaseGVN* phase, bool can_reshape);
 };
 
+
+//-----------------------------ReinterpretS2HFNode ---------------------------
+// Reinterpret Short to Half Float
+class ReinterpretS2HFNode : public Node {
+  public:
+  ReinterpretS2HFNode(Node* in1) : Node(0, in1) {}
```

Comment:
```suggestion
  ReinterpretS2HFNode(Node* in1) : Node(nullptr, in1) {}
```
Oh, just caught this. I think you should not use `0` here any more, check all other uses.